src/gluonts/ext/r_forecast/_predictor.py

```diff
@@ -104,21 +105,17 @@ def __init__(
 
         self._robjects = robjects
         self._rinterface = rinterface
+        self._rcallbacks = callbacks
         self._rinterface.initr()
         self._rpackages = rpackages
 
-        this_dir = os.path.dirname(os.path.realpath(__file__))
-        this_dir = this_dir.replace("\\", "/")  # for windows
-        r_files = [
-            n[:-2]
-            for n in os.listdir(f"{this_dir}/R/")
-            if n[-2:] == ".R" and n.startswith(r_file_prefix)
-        ]
+        this_dir = Path(__file__).resolve().parent.absolute()
+        this_dir = Path(f"{this_dir}".replace("\\", "/"))  # for windows
```

Comment:
I'm curious, is this because R always expects `/`?

Reply:
Not sure what's going on here. I kept the logic same as before, just changed `os.path` stuff to `pathlib.Path`.